orchagent/vnetorch.h

```diff
@@ -61,7 +61,7 @@ extern std::vector<VR_TYPE> vr_cntxt;
 class VNetRequest : public Request
 {
 public:
-    VNetRequest() : Request(vnet_request_description, ':') { }
+    VNetRequest() : Request(vnet_request_description, '|') { }
```

Comment:
@liron-ze ,` | `is used as a delimiter in config_db. Can you keep the same `:` and try fixing this in `Request `class?